polymer.externs.js

```diff
@@ -53,11 +53,11 @@ PolymerElement.prototype.is;
 PolymerElement.prototype.extends;
 
 /**
- * An array of objects whose properties get mixed in to this element.
+ * An array of objects whose properties get added to this element.
```

Comment:
Maybe just add an @see to https://www.polymer-project.org/1.0/docs/devguide/behaviors.html